packages/psss/src/containers/Tables/FakeEventBasedDataTable.js

```diff
@@ -0,0 +1,58 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+import { Table } from '@tupaia/ui-components';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useAlerts } from '../../api';
+import { AlertMenuCell, SyndromeCell, WeekAndDateCell } from '../../components';
+import { getCurrentPeriod } from '../../utils';
+
+const columns = [
+  {
+    title: 'Syndrome',
+    key: 'syndrome',
+    align: 'left',
+    CellComponent: SyndromeCell,
+  },
+  {
+    title: 'Alert Start Date',
+    key: 'period',
+    align: 'left',
+    width: '200px',
+    CellComponent: WeekAndDateCell,
+  },
+  {
+    title: 'Cases Since Alert Began',
+    key: 'totalCases',
+    align: 'left',
+    width: '115px',
+  },
+  {
+    title: '',
+    key: 'id',
+    sortable: false,
+    CellComponent: AlertMenuCell,
+    width: '45px',
+  },
+];
+
+export const FakeEventBasedDataTable = ({ countryCode }) => {
```

Comment:
Code here is shamelessly copy-pasted from [AlertsTable.js](https://github.com/beyondessential/tupaia/blob/psss-phase-2/packages/psss/src/containers/Tables/AlertsTable.js), with a few tweaks.

I could instead update `AlertsTable` to accommodate the placeholder's needs, and that wouldn't require many changes. However, I decided it's still not worth doing so just for the needs of a temporary part of the codebase.

Reply:
SHAME! 😝